import-export-cli/cmd/importAPI.go

```diff
@@ -82,60 +70,21 @@ var ImportAPICmd = &cobra.Command{
 	Example: importAPICmdExamples,
 	Run: func(cmd *cobra.Command, args []string) {
 		utils.Logln(utils.LogPrefixInfo + importAPICmdLiteral + " called")
-		var apisExportDirectory = filepath.Join(utils.ExportDirectory, utils.ExportedApisDirName)
-
-		cred, err := getCredentials(importEnvironment)
+		credential, err := getCredentials(importEnvironment)
```

Comment:
Shall we use as credentials?

Reply:
Used `cred` as its the variable name used in all the other places.